.github/workflows/release-github.yml

```diff
@@ -37,3 +37,12 @@ jobs:
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  release-to-winget:
+    needs: [ build ]
+    runs-on: windows-latest
+    steps:
+      - name: Release to Winget
+        uses: vedantmgoyal2009/winget-releaser@v2
```

Comment:
This releaser requires forking https://github.com/microsoft/winget-pkgs to `oras-project`. It seems not right.

Reply:
What are the proper steps to release a package to winget manually?